src/user/api/endpoints.ts

```diff
@@ -1,3 +1,4 @@
 export const endpoints = {
   UPDATE_MY_PASSWORD: '/api/user/password/change',
+  RESET_PASSWORD: '/api/user/password/forgot-validate',
```

Comment:
```suggestion
  RESET_PASSWORD_VALIDATE: '/api/user/password/forgot-validate',
```

On pourrait avoir un nommage plus précis pour comprendre que ça doit faire suite à la requête de changement de mot de passe.